BackendCommands.php

```diff
@@ -134,43 +118,170 @@ public function updateCode()
+        $this->populateConfigSyncDirectory();
     }
 
     /**
      * Export configuration of BurdaStyle backend site.
      *
+     * TODO: Revisit when we do not need the local environment config folder anymore.
+     * TODO: Then decide if we can make this a post command for config:export.
+     *
      * @command backend:config-export
      *
      * @aliases backend:cex
      *
      * @options-backend
      *
-     * @validate-site-alias
-     *
-     * @usage drush @elle backend:export-config
+     * @usage drush @elle backend:config-export
      *   Exports the elle configuration.
      *
      * @bootstrap config
      */
     public function configExport()
     {
+        if ($this->environment !== 'prod') {
+            $this->logger()->error('Only production will be exported. Current environment is "%environment".', ['%environment' => $this->environment]);
+
+            return;
+        }
+
         // export the config into the export folder.
-        $this->drush($this->selfRecord(), 'config:export', [], ['yes' => true]);
+        $this->drush($this->selfRecord(), 'config:export', [], ['yes' => $this->input()->getOption('yes')]);
+    }
 
-        if ($this->environment !== 'local') {
-            // Nimbus will overwrite local config files with the production values.
-            $this->process(['git', 'checkout', $this->siteConfigSyncDirectory().'/../local']);
+    /**
+     * Move files from sync folder to shared or override folders.
+     *
+     * As long as we have to handle environment specific config, this can not
+     * be a post command for the default drush config:import command.
+     *
+     * TODO: Revisit when we do not need the local environment config folder anymore.
+     * TODO: Then decide if we can make this a post command for config:export.
+     *
+     * @hook post-command backend:config-export
+     *
+     * @param $result
+     * @param \Consolidation\AnnotatedCommand\CommandData $commandData
+     */
+    public function postConfigExportCommand($result, CommandData $commandData)
+    {
+        $exportedFiles = $this->getConfigFilesInDirectory($this->siteConfigSyncDirectory());
+        $overrideFiles = $this->getConfigFilesInDirectory($this->siteConfigOverrideDirectory());
+        $sharedFiles = $this->getConfigFilesInDirectory($this->configSharedDirectory());
+        $localFiles = $this->getConfigFilesInDirectory($this->siteConfigEnvironmentDirectory('local'));
+        $modifiedFiles = [];
+
+        foreach ($exportedFiles as $fileName => $fullPath) {
+            // First check, if the file should be put into the override directory.
+            // otherwise check, if the file should be put into the shared directory.
+            // Finally, if file is new (neither in shared nor in override),
+            // put it into override and inform user of new config.
+            if (isset($overrideFiles[$fileName])) {
+                if (!$this->filesAreEqual($overrideFiles[$fileName], $fullPath)) {
```

Comment:
I was thinking, maybe we could use the [StorageComparer](https://git.drupalcode.org/project/drupal/-/blob/9.2.x/core/lib/Drupal/Core/Config/StorageComparer.php) instead, similar to [config:export](https://github.com/drush-ops/drush/blob/843dfb1e449ac7f5c881175ea173b7b3fa5094b6/src/Drupal/Commands/config/ConfigExportCommands.php#L136).
But that's a follow-up, I guess

Reply:
PLease write am issue for that